modules/ROOT/pages/user-management.adoc

```diff
@@ -236,3 +240,14 @@ You can select the project(s) you have been invited to and choose to accept or d
 ====
 User management within the Aura console does not replace built-in roles or fine-grained RBAC at the database level.
 ====
+
+== Metrics reader role
+
+The `metrics reader` role can be assigned to any user,
```

Comment:
Maybe add something about service accounts and downplay the assignment to _users_?